docs/recipes/running-tasks-in-series.md

```diff
@@ -1,66 +1,95 @@
-# Running tasks in series, i.e. Task Dependency
+# Running tasks in series
 
-By default, tasks run with maximum concurrency -- e.g. it launches all the tasks at once and waits for nothing. If you want to create a series where tasks run in a particular order, you need to do two things:
+By default, gulp cli run task with maximum concurrency -- e.g. it launches all the tasks at once 
```

Comment:
cli -> CLI?

"--" -> "—"?

Why do you have a line break at 96 characters? lol